pkgs/development/python-modules/dictdiffer/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, check-manifest
+, mock
+, pycodestyle
+, pydocstyle
+, pytest
+, pytest-cov
+, pytest-pycodestyle
+, pytest-pydocstyle
+, pytest-isort
+, pytest-runner
```

Comment:
```suggestion
```
Please patch out the usage of pytest-runner. It does nothing.

Reply:
You say, patch the upstream source code?

Reply:
Yes, pytest-runner does not fully work in nixpkgs anyway and we normally patch the requirement on it out together with pytest-cov.